go/test/endtoend/cluster/vtctldclient_process.go

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2019 The Vitess Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+	"time"
+
+	"vitess.io/vitess/go/vt/log"
+)
+
+// VtctldClientProcess is a generic handle for a running vtctldclient command .
```

Comment:
thank you for doing this!! i've been meaning to do this forever now, thank you so much!!

Reply:
fixed